docs/install/yaml-install/eventing/eventing-installation-files.md

```diff
@@ -12,10 +12,10 @@ The following table describes the installation files included in Knative Eventin
 
 | File name | Description | Dependencies|
 | --- | --- | --- |
-| eventing-core.yaml | Required: Knative Eventing core components. |  eventing-crds.yaml |
-| eventing-crds.yaml | Required: Knative Eventing core CRDs. |  none |
-| eventing-post-install.yaml | Jobs required for upgrading to a new minor version. | eventing-core.yaml, eventing-crds.yaml |
-| eventing-sugar-controller.yaml | Reconciler that watches for labels and annotations on certain resources to inject eventing components. | eventing-core.yaml |
-| eventing.yaml | Combines `eventing-core.yaml`, `mt-channel-broker.yaml`, and `in-memory-channel.yaml`. | none |
-| in-memory-channel.yaml | Components to configure In-Memory Channels. | eventing-core.yaml |
-| mt-channel-broker.yaml | Components to configure Multi-Tenant (MT) Channel Broker. | eventing-core.yaml |
+| [eventing-core.yaml]({{ artifact(repo="eventing",file="eventing-core.yaml")}}) | Required: Knative Eventing core components. |  [eventing-crds.yaml]({{ artifact(repo="eventing",file="eventing-crds.yaml")}}) |
+| [eventing-crds.yaml]({{ artifact(repo="eventing",file="eventing-crds.yaml")}}) | Required: Knative Eventing core CRDs. |  none |
+| [eventing-post-install.yaml]({{ artifact(repo="eventing",file="eventing-post-install.yaml")}}) | Jobs required for upgrading to a new minor version. | [eventing-core.yaml]({{ artifact(repo="eventing",file="eventing-core.yaml")}}), [eventing-crds.yaml]({{ artifact(repo="eventing",file="eventing-crds.yaml")}}) |
+| [eventing-sugar-controller.yaml]({{ artifact(repo="eventing",file="eventing-sugar-controller.yaml")}}) | Reconciler that watches for labels and annotations on certain resources to inject eventing components. | [eventing-core.yaml]({{ artifact(repo="eventing",file="eventing-core.yaml")}}) |
```

Comment:
@pierDipi @creydr is this still used? The link seems to end up in 404.

Reply:
IIRC the sugar controller can be enabled now via a config map and no additional deployment is required. Anyhow I leave it to @pierDipi for the final confirmation, that this line can be removed

Reply:
https://deploy-preview-5751--knative.netlify.app/docs/eventing/configuration/sugar-configuration/

Reply:
If so, we should update at least two files in addition to the one above:
- docs/docs/install/uninstall.md -> [L148-L154](https://github.com/knative/docs/blob/adf7258d2939851af8810b60c7e5c9f168ef32b3/docs/install/uninstall.md?plain=1#L148-L154)
- docs/docs/install/yaml-install/eventing/install-eventing-with-yaml.md -> [L231-L254](https://github.com/knative/docs/blob/adf7258d2939851af8810b60c7e5c9f168ef32b3/docs/install/yaml-install/eventing/install-eventing-with-yaml.md?plain=1#L231-L254)

Do you think there are others?

Reply:
@pierDipi Should I create a new PR with the changes mentioned above?